docs/request.md

```diff
@@ -202,23 +203,23 @@ Request URLs may contain interpolations:
 
 ```javascript
 m.request({
-	method: "GET",
-	url: "/api/v1/users/:id",
-	data: {id: 123},
+  method: "GET",
+  url: "/api/v1/users/:id",
+  params: {id: 123},
 }).then(function(user) {
-	console.log(user.id) // logs 123
+  console.log(user.id) // logs 123
 })
 ```
 
 In the code above, `:id` is populated with the data from the `{id: 123}` object, and the request becomes `GET /api/v1/users/123`.
```

Comment:
```suggestion
In the code above, `:id` is populated with the data from the `params` object, and the request becomes `GET /api/v1/users/123`.
```

Reply:
This line needs fixed (and the above suggestion would work).

(Including this purely for the sake of review.)

Reply:
OK